streampipes-service-core/src/main/java/org/apache/streampipes/service/core/scheduler/DataLakeScheduler.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.streampipes.service.core.scheduler;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+
+import org.slf4j.LoggerFactory;
+import org.apache.streampipes.dataexplorer.api.IDataExplorerQueryManagement;
+import org.apache.streampipes.dataexplorer.api.IDataExplorerSchemaManagement;
+import org.apache.streampipes.dataexplorer.management.DataExplorerDispatcher;
+import org.apache.streampipes.model.datalake.DataLakeMeasure;
+import org.slf4j.Logger;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataLakeScheduler {
+
+    private static final Logger log = LoggerFactory.getLogger(DataLakeScheduler.class);
```

Comment:
```suggestion
    private static final Logger LOG = LoggerFactory.getLogger(DataLakeScheduler.class);
```

In most cases we use LOG as the variable for logging.